avm/utilities/pipelines/e2eValidation/resourceRemoval/helper/Invoke-ResourcePostRemoval.ps1

```diff
@@ -18,140 +18,140 @@ Purge the resource 'myVault' of type 'Microsoft.KeyVault/vaults' with ID '/subsc
 #>
 function Invoke-ResourcePostRemoval {
 
-    [CmdletBinding(SupportsShouldProcess)]
-    param (
-        [Parameter(Mandatory = $true)]
-        [string] $ResourceId,
-
-        [Parameter(Mandatory = $true)]
-        [string] $Type
-    )
-
-    switch ($Type) {
-        'Microsoft.AppConfiguration/configurationStores' {
-            $subscriptionId = $ResourceId.Split('/')[2]
-            $resourceName = Split-Path $ResourceId -Leaf
-
-            # Fetch service in soft-delete
-            $getPath = '/subscriptions/{0}/providers/Microsoft.AppConfiguration/deletedConfigurationStores?api-version=2021-10-01-preview' -f $subscriptionId
-            $getRequestInputObject = @{
-                Method = 'GET'
-                Path   = $getPath
-            }
-            $softDeletedConfigurationStore = ((Invoke-AzRestMethod @getRequestInputObject).Content | ConvertFrom-Json).value | Where-Object { $_.properties.configurationStoreId -eq $ResourceId }
-
-            if ($softDeletedConfigurationStore) {
-                # Purge service
-                $purgePath = '/subscriptions/{0}/providers/Microsoft.AppConfiguration/locations/{1}/deletedConfigurationStores/{2}/purge?api-version=2021-10-01-preview' -f $subscriptionId, $softDeletedConfigurationStore.properties.location, $resourceName
-                $purgeRequestInputObject = @{
-                    Method = 'POST'
-                    Path   = $purgePath
-                }
-                Write-Verbose ('[*] Purging resource [{0}] of type [{1}]' -f $resourceName, $Type) -Verbose
-                if ($PSCmdlet.ShouldProcess(('App Configuration Store with ID [{0}]' -f $softDeletedConfigurationStore.properties.configurationStoreId), 'Purge')) {
-                    $response = Invoke-AzRestMethod @purgeRequestInputObject
-                    if ($response.StatusCode -ne 200) {
-                        throw ('Purge of resource [{0}] failed with error code [{1}]' -f $ResourceId, $response.StatusCode)
-                    }
-                }
-            }
-            break
+  [CmdletBinding(SupportsShouldProcess)]
```

Comment:
Hey @sebassem, any idea why your VSCode instance did not pick up the default vscode settings? The define tab-size is 4, yet it seems it was reformatted to 2 in this PR.

Reply:
Also, was there any update to the file aside from whitespaces? It's a bit hard to tell as it shows so many changes due to the whitespaces 😄